新版本提示AttributeError: ("'NoneType' object has no attribute 'get_live_action'", [])

## 环境

- nonebot-bison 版本：fix/bilibili-dyn
- nonebot 版本：2.2.1
- 安装方式：（以下方式的一种或者其他方式）
  1. 通过 nb-cli 安装

- 操作系统：WIN10

## 问题

显示报错("'NoneType' object has no attribute 'get_live_action'", [])

## 日志

```

05-04 20:11:20 [ERROR] apscheduler | Job "Scheduler.exec_fetch (trigger: interval[0:00:15], next run at: 2024-05-04 20:11:35 CST)" raised an exception
Traceback (most recent call last):
  File "<string>", line 15, in <module>
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\__init__.py", line 334, in run
    get_driver().run(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\main.py", line 575, in run
    server.run()
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "C:\Users\\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
  File "C:\Users\\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1906, in _run_once
    handle._run()
  File "C:\Users\\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
 File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\scheduler\scheduler.py", line 107, in exec_fetch
    to_send = await platform_obj.do_fetch_new_post(SubUnit(schedulable.target, send_userinfo_list))
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 109, in do_fetch_new_post
    return await catch_network_error(self.fetch_new_post, sub_unit) or []
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 53, in catch_network_error
    return await func(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 396, in fetch_new_post
    return await self._handle_status_change(new_status, sub_unit)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 376, in _handle_status_change
    diff = self.compare_status(sub_unit.sub_target, old_status, new_status)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\bilibili\platforms.py", line 443, in compare_status
    match new_status.get_live_action(old_status):
AttributeError: ("'NoneType' object has no attribute 'get_live_action'", [])
```

- [√] 我搜索过了 issue，但是并没有发现过与我类似的问题
- [√] 我确认在日志中去掉了敏感信息

以及新的2个报错log，导致直播无法推送，服务器规则代理日本
```05-14 08:16:14 [WARNING] nonebot_bison | API request record: https://api.live.bilibili.com/room/v1/Room/get_status_info_by_uids?uids%5B%5D=1827139579&uids%5B%5D=666726799&uids%5B%5D=14950589&uids%5B%5D=1873291&uids%5B%5D=690608698&uids%5B%5D=477317922&uids%5B%5D=1464029182&uids%5B%5D=14387072&uids%5B%5D=1542516095&uids%5B%5D=1762745&uids%5B%5D=2039332008&uids%5B%5D=617459493&uids%5B%5D=863287&uids%5B%5D=594827452&uids%5B%5D=1950658&uids%5B%5D=56748733&uids%5B%5D=731556&uids%5B%5D=23269266&uids%5B%5D=1873291%20 Headers({'host': 'api.live.bilibili.com', 'accept': '*/*', 'accept-encoding': 'gzip, deflate', 'connection': 'keep-alive', 'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36', 'cookie':  [200] Headers({'date': 'Tue, 14 May 2024 00:16:14 GMT', 'content-type': 'application/json', 'content-length': '143', 'connection': 'keep-alive', 'bili-status-code': '65530', 'bili-trace-id': '64cb2cb9a513a3e90ada9038056642ad:17cf32bde09505fd:ada9038056642ad:0', 'cache-control': 'no-cache', 'server': 'swoole-http-server', 'x-bili-trace-id': '64cb2cb9a513a3e90ada9038056642ad:17cf32bde09505fd:ada9038056642ad:0', 'x-envoy-upstream-service-time': '0', 'vary': 'Accept-Encoding,Origin', 'expires': 'Tue, 14 May 2024 00:16:13 GMT', 'x-cache-webcdn': 'BYPASS from blzone06'}) {"code":65530,"msg":"route_exception","message":"route_exception","data":{"excp_code":0,"excp_msg":"Expect integer.","excp_class":"Exception"}}
05-14 08:16:14 [ERROR] apscheduler | Job "Scheduler.exec_fetch (trigger: interval[0:00:15], next run at: 2024-05-14 08:16:28 CST)" raised an exception
Traceback (most recent call last):
  File "<string>", line 15, in <module>
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\__init__.py", line 334, in run
    get_driver().run(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\main.py", line 575, in run
    server.run()
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1906, in _run_once
    handle._run()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
> File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\scheduler\scheduler.py", line 102, in exec_fetch
    to_send = await platform_obj.do_batch_fetch_new_post(sub_units)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 115, in do_batch_fetch_new_post
    return await catch_network_error(self.batch_fetch_new_post, sub_units) or []
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 53, in catch_network_error
    return await func(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 401, in batch_fetch_new_post
    new_statuses = await self.batch_get_status([x[0] for x in sub_units])
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\bilibili\platforms.py", line 430, in batch_get_status
    raise self.FetchError()
nonebot_bison.platform.platform.StatusChange.FetchError: ['https://api.live.bilibili.com/room/v1/Room/get_status_info_by_uids?uids%5B%5D=1827139579&uids%5B%5D=666726799&uids%5B%5D=14950589&uids%5B%5D=1873291&uids%5B%5D=690608698&uids%5B%5D=477317922&uids%5B%5D=1464029182&uids%5B%5D=14387072&uids%5B%5D=1542516095&uids%5B%5D=1762745&uids%5B%5D=2039332008&uids%5B%5D=617459493&uids%5B%5D=863287&uids%5B%5D=594827452&uids%5B%5D=1950658&uids%5B%5D=56748733&uids%5B%5D=731556&uids%5B%5D=23269266&uids%5B%5D=1873291%20 Headers({\'host\': \'api.live.bilibili.com\', \'accept\': \'*/*\', \'accept-encoding\': \'gzip, deflate\', \'connection\': \'keep-alive\', \'user-agent\': \'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36\', \'cookie\': | [200] Headers({\'date\': \'Tue, 14 May 2024 00:16:14 GMT\', \'content-type\': \'application/json\', \'content-length\': \'143\', \'connection\': \'keep-alive\', \'bili-status-code\': \'65530\', \'bili-trace-id\': \'64cb2cb9a513a3e90ada9038056642ad:17cf32bde09505fd:ada9038056642ad:0\', \'cache-control\': \'no-cache\', \'server\': \'swoole-http-server\', \'x-bili-trace-id\': \'64cb2cb9a513a3e90ada9038056642ad:17cf32bde09505fd:ada9038056642ad:0\', \'x-envoy-upstream-service-time\': \'0\', \'vary\': \'Accept-Encoding,Origin\', \'expires\': \'Tue, 14 May 2024 00:16:13 GMT\', \'x-cache-webcdn\': \'BYPASS from blzone06\'}) {"code":65530,"msg":"route_exception","message":"route_exception","data":{"excp_code":0,"excp_msg":"Expect integer.","excp_class":"Exception"}}']
```

```05-14 08:16:44 [WARNING] nonebot_bison | API request record: https://api.live.bilibili.com/room/v1/Room/get_status_info_by_uids?uids%5B%5D=1827139579&uids%5B%5D=666726799&uids%5B%5D=14950589&uids%5B%5D=1873291&uids%5B%5D=690608698&uids%5B%5D=477317922&uids%5B%5D=1464029182&uids%5B%5D=14387072&uids%5B%5D=1542516095&uids%5B%5D=1762745&uids%5B%5D=2039332008&uids%5B%5D=617459493&uids%5B%5D=863287&uids%5B%5D=594827452&uids%5B%5D=1950658&uids%5B%5D=56748733&uids%5B%5D=731556&uids%5B%5D=23269266&uids%5B%5D=1873291%20 Headers({'host': 'api.live.bilibili.com', 'accept': '*/*', 'accept-encoding': 'gzip, deflate', 'connection': 'keep-alive', 'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36', 'cookie':  [200] Headers({'date': 'Tue, 14 May 2024 00:16:44 GMT', 'content-type': 'application/json', 'content-length': '143', 'connection': 'keep-alive', 'bili-status-code': '65530', 'bili-trace-id': '1ddf2d1c3b4e6dd867a9dbface6642ad:17cf32c4ddb4183b:67a9dbface6642ad:0', 'cache-control': 'no-cache', 'server': 'swoole-http-server', 'x-bili-trace-id': '1ddf2d1c3b4e6dd867a9dbface6642ad:17cf32c4ddb4183b:67a9dbface6642ad:0', 'x-envoy-upstream-service-time': '1', 'vary': 'Accept-Encoding,Origin', 'expires': 'Tue, 14 May 2024 00:16:43 GMT', 'x-cache-webcdn': 'BYPASS from blzone05'}) {"code":65530,"msg":"route_exception","message":"route_exception","data":{"excp_code":0,"excp_msg":"Expect integer.","excp_class":"Exception"}}
05-14 08:16:44 [ERROR] apscheduler | Job "Scheduler.exec_fetch (trigger: interval[0:00:15], next run at: 2024-05-14 08:16:58 CST)" raised an exception
Traceback (most recent call last):
  File "<string>", line 15, in <module>
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\__init__.py", line 334, in run
    get_driver().run(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\main.py", line 575, in run
    server.run()
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
    self.run_forever()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
    self._run_once()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1906, in _run_once
    handle._run()
  File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
> File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\scheduler\scheduler.py", line 102, in exec_fetch
    to_send = await platform_obj.do_batch_fetch_new_post(sub_units)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 115, in do_batch_fetch_new_post
    return await catch_network_error(self.batch_fetch_new_post, sub_units) or []
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 53, in catch_network_error
    return await func(*args, **kwargs)
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\platform.py", line 401, in batch_fetch_new_post
    new_statuses = await self.batch_get_status([x[0] for x in sub_units])
  File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\platform\bilibili\platforms.py", line 430, in batch_get_status
    raise self.FetchError()
nonebot_bison.platform.platform.StatusChange.FetchError: ['https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/space?host_mid=&timezone_offset=-480&offset= Headers({\'host\': \'api.bilibili.com\', \'accept\': \'*/*\', \'accept-encoding\': \'gzip, deflate\', \'connection\': \'keep-alive\', \'user-agent\': \'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36\', \'cookie\| [200] Headers({\'date\': \'Tue, 14 May 2024 00:16:44 GMT\', \'content-type\': \'application/json; charset=utf-8\', \'transfer-encoding\': \'chunked\', \'connection\': \'keep-alive\', \'bili-status-code\': \'0\', \'bili-trace-id\': \'25fcbb4f2d6642ad\', \'cpu_usage\': \'13\', \'x-bili-trace-id\': \'18ac1b70f5d9563e25fcbb4f2d6642ad\', \'x-rid-result\': \'2\', \'content-encoding\': \'gzip\', \'expires\': \'Tue, 14 May 2024 00:16:43 GMT\', \'cache-control\': \'no-cache\', \'x-cache-webcdn\': \'BYPASS from blzone05\'}) {"count":1,"forbidden":false,"status":false}}},"type":"DYNAMIC_TYPE_AV","visible":true}],"offset":"874585122531180563","update_baseline":"","update_num":0}}', 'https://api.live.bilibili.com/room/v1/Room/get_status_info_by_uids?uids%5B%5D=1827139579&uids%5B%5D=666726799&uids%5B%5D=14950589&uids%5B%5D=1873291&uids%5B%5D=690608698&uids%5B%5D=477317922&uids%5B%5D=1464029182&uids%5B%5D=14387072&uids%5B%5D=1542516095&uids%5B%5D=1762745&uids%5B%5D=2039332008&uids%5B%5D=617459493&uids%5B%5D=863287&uids%5B%5D=594827452&uids%5B%5D=1950658&uids%5B%5D=56748733&uids%5B%5D=731556&uids%5B%5D=23269266&uids%5B%5D=1873291%20 Headers({\'host\': \'api.live.bilibili.com\', \'accept\': \'*/*\', \'accept-encoding\': \'gzip, deflate\', \'connection\': \'keep-alive\', \'user-agent\': \'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36\', \'cookie\'| [200] Headers({\'date\': \'Tue, 14 May 2024 00:16:44 GMT\', \'content-type\': \'application/json\', \'content-length\': \'143\', \'connection\': \'keep-alive\', \'bili-status-code\': \'65530\', \'bili-trace-id\': \'1ddf2d1c3b4e6dd867a9dbface6642ad:17cf32c4ddb4183b:67a9dbface6642ad:0\', \'cache-control\': \'no-cache\', \'server\': \'swoole-http-server\', \'x-bili-trace-id\': \'1ddf2d1c3b4e6dd867a9dbface6642ad:17cf32c4ddb4183b:67a9dbface6642ad:0\', \'x-envoy-upstream-service-time\': \'1\', \'vary\': \'Accept-Encoding,Origin\', \'expires\': \'Tue, 14 May 2024 00:16:43 GMT\', \'x-cache-webcdn\': \'BYPASS from blzone05\'}) {"code":65530,"msg":"route_exception","message":"route_exception","data":{"excp_code":0,"excp_msg":"Expect integer.","excp_class":"Exception"}}']
```

```
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for arknights with interval {'seconds': 30}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for bilibili.com with interval {'seconds': 40}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for live.bilibili.com with interval {'seconds': 15}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for AnonymousScheduleConfig[ff14] with interval {'seconds': 60}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for music.163.com with interval {'minutes': 1}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for rss with interval {'seconds': 30}
05-14 08:29:52 [INFO] nonebot_bison | register scheduler for weibo.com with interval {'seconds': 3}
```

遇到了一样的问题，报错信息AttributeError: ("'NoneType' object has no attribute 'get_live_action'", [])以下是关注列表
B站 | 弥希Miki | 477317922 | 一般动态 专栏文章 视频 纯文字 转发
Bilibili直播 | 弥希Miki | 477317922 | 开播提醒 标题更新提醒
新浪微博 | 弥希MIKI | 7576338081 | 转发 视频 图文 文字
B站 | 千幽Chiyuu | 690608691 | 一般动态 专栏文章 视频 纯文字 转发
Bilibili直播 | 千幽Chiyuu | 690608691 | 开播提醒 标题更新提醒

**日志**
```
AttributeError: ("'NoneType' object has no attribute 'get_live_action'", [])
06-15 03:38:57 [ERROR] apscheduler | Job "Scheduler.exec_fetch (trigger: interval[0:00:03], next run at: 2024-06-15 03:39:00 CST)" raised an exception
Traceback (most recent call last):
  File "<string>", line 21, in <module>
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot\__init__.py", line 335, in run
    get_driver().run(*args, **kwargs)
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
    uvicorn.run(
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\uvicorn\main.py", line 577, in run
    server.run()
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\uvicorn\server.py", line 65, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 190, in run
    return runner.run(main)
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\runners.py", line 118, in run
    return self._loop.run_until_complete(task)
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 641, in run_until_complete
    self.run_forever()
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 608, in run_forever
    self._run_once()
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\base_events.py", line 1936, in _run_once
    handle._run()
  File "C:\Users\***\AppData\Local\Programs\Python\Python311\Lib\asyncio\events.py", line 84, in _run
    self._context.run(self._callback, *self._args)
> File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\apscheduler\executors\base_py3.py", line 30, in run_coroutine_job
    retval = await job.func(*job.args, **job.kwargs)
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\scheduler\scheduler.py", line 107, in exec_fetch
    to_send = await platform_obj.do_fetch_new_post(SubUnit(schedulable.target, send_userinfo_list))
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\platform\platform.py", line 109, in do_fetch_new_post
    return await catch_network_error(self.fetch_new_post, sub_unit) or []
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\platform\platform.py", line 53, in catch_network_error
    return await func(*args, **kwargs)
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\platform\platform.py", line 396, in fetch_new_post
    return await self._handle_status_change(new_status, sub_unit)
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\platform\platform.py", line 376, in _handle_status_change
    diff = self.compare_status(sub_unit.sub_target, old_status, new_status)
  File "C:\Users\***\Desktop\QQbot\nonebot\***\.venv\Lib\site-packages\nonebot_bison\platform\bilibili.py", line 449, in compare_status
    match new_status.get_live_action(old_status):
AttributeError: ("'NoneType' object has no attribute 'get_live_action'", [])
```

```08-08 16:51:18 [ERROR] uvicorn | Exception in ASGI application
﻿
Traceback (most recent call last):
File "<string>", line 15, in <module>
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\__init__.py", line 335, in run
get_driver().run(*args, **kwargs)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot\drivers\fastapi.py", line 186, in run
uvicorn.run(
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\main.py", line 575, in run
server.run()
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\server.py", line 65, in run
return asyncio.run(self.serve(sockets=sockets))
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\runners.py", line 44, in run
return loop.run_until_complete(main)
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 636, in run_until_complete
self.run_forever()
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\windows_events.py", line 321, in run_forever
super().run_forever()
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 603, in run_forever
self._run_once()
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 1906, in _run_once
handle._run()
File "C:\Users\550N\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
self._context.run(self._callback, *self._args)
> File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\protocols\http\httptools_impl.py", line 411, in run_asgi
result = await app(  # type: ignore[func-returns-value]
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\uvicorn\middleware\proxy_headers.py", line 69, in __call__
return await self.app(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\fastapi\applications.py", line 1054, in __call__
await super().__call__(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\applications.py", line 123, in __call__
await self.middleware_stack(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\errors.py", line 186, in __call__
raise exc
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\errors.py", line 164, in __call__
await self.app(scope, receive, _send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\exceptions.py", line 65, in __call__
await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 64, in wrapped_app
raise exc
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
await app(scope, receive, sender)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 756, in __call__
await self.middleware_stack(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 776, in app
await route.handle(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 485, in handle
await self.app(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\fastapi\applications.py", line 1054, in __call__
await super().__call__(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\applications.py", line 123, in __call__
await self.middleware_stack(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\errors.py", line 186, in __call__
raise exc
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\errors.py", line 164, in __call__
await self.app(scope, receive, _send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\middleware\exceptions.py", line 65, in __call__
await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 64, in wrapped_app
raise exc
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
await app(scope, receive, sender)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 756, in __call__
await self.middleware_stack(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 776, in app
await route.handle(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 297, in handle
await self.app(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 77, in app
await wrap_app_handling_exceptions(app, request)(scope, receive, send)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 64, in wrapped_app
raise exc
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
await app(scope, receive, sender)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\starlette\routing.py", line 72, in app
response = await func(request)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\fastapi\routing.py", line 278, in app
raw_response = await run_endpoint_function(
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\fastapi\routing.py", line 191, in run_endpoint_function
return await dependant.call(**values)
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\admin_page\api.py", line 144, in get_target_name
return {"targetName": await check_sub_target(platformName, T_Target(target))}
File "D:\BOT\Nonbot2\Bison\.venv\lib\site-packages\nonebot_bison\apis.py", line 9, in check_sub_target
scheduler = scheduler_dict[scheduler_conf_class]
KeyError: <class 'nonebot_bison.platform.bilibili.scheduler.BilibiliSite'>
```
新版本添加B站用户提示这个

--- a/nonebot_bison/platform/__init__.py
+++ b/nonebot_bison/platform/__init__.py
@@ -3,6 +3,7 @@ from pkgutil import iter_modules
 from collections import defaultdict
 from importlib import import_module
 
+from ..plugin_config import plugin_config
 from .platform import Platform, make_no_target_group
 
 _package_dir = str(Path(__file__).resolve().parent)
@@ -22,3 +23,15 @@ for name, platform_list in _platform_list.items():
         platform_manager[name] = platform_list[0]
     else:
         platform_manager[name] = make_no_target_group(platform_list)
+
+
+def _get_unavailable_platforms() -> dict[str, str]:
+    res = {}
+    for name, platform in platform_manager.items():
+        if platform.site.require_browser and not plugin_config.bison_use_browser:
+            res[name] = "需要启用 bison_use_browser"
+    return res
+
+
+# platform => reason for not available
+unavailable_paltforms: dict[str, str] = _get_unavailable_platforms()

--- a/nonebot_bison/sub_manager/add_sub.py
+++ b/nonebot_bison/sub_manager/add_sub.py
@@ -9,9 +9,9 @@ from nonebot_plugin_saa import Text, PlatformTarget, SupportedAdapters
 from ..types import Target
 from ..config import config
 from ..apis import check_sub_target
-from ..platform import Platform, platform_manager
 from ..config.db_config import SubscribeDupException
 from .utils import common_platform, ensure_user_info, gen_handle_cancel
+from ..platform import Platform, platform_manager, unavailable_paltforms
 
 
 def do_add_sub(add_sub: type[Matcher]):
@@ -39,6 +39,8 @@ def do_add_sub(add_sub: type[Matcher]):
         elif platform == "取消":
             await add_sub.finish("已中止订阅")
         elif platform in platform_manager:
+            if platform in unavailable_paltforms:
+                await add_sub.finish(f"无法订阅 {platform}，{unavailable_paltforms[platform]}")
             state["platform"] = platform
         else:
             await add_sub.reject("平台输入错误")

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -18,6 +18,7 @@ def pytest_configure(config: pytest.Config) -> None:
         "superusers": {"10001"},
         "command_start": {""},
         "log_level": "TRACE",
+        "bison_use_browser": True,
     }
 
 

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -113,3 +113,12 @@ async def use_legacy_config(app: App):
 
     # 清除单例的缓存
     Singleton._instances.clear()
+
+
+@pytest.fixture
+async def _no_browser(app: App, mocker: MockerFixture):
+    from nonebot_bison.plugin_config import plugin_config
+    from nonebot_bison.platform import _get_unavailable_platforms
+
+    mocker.patch.object(plugin_config, "bison_use_browser", False)
+    mocker.patch("nonebot_bison.platform.unavailable_paltforms", _get_unavailable_platforms())

--- a/tests/sub_manager/test_add.py
+++ b/tests/sub_manager/test_add.py
@@ -615,3 +615,48 @@ async def test_add_with_bilibili_bangumi_target_parser(app: App, init_scheduler)
     assert sub.tags == []
     assert sub.target.platform_name == "bilibili-bangumi"
     assert sub.target.target_name == "汉化日记 第三季"
+
+
+@pytest.mark.asyncio
+async def test_subscribe_platform_requires_browser(app: App, mocker: MockerFixture):
+    from nonebot.adapters.onebot.v11.event import Sender
+    from nonebot.adapters.onebot.v11.message import Message
+
+    from nonebot_bison.plugin_config import plugin_config
+    from nonebot_bison.sub_manager import add_sub_matcher, common_platform
+    from nonebot_bison.platform import platform_manager, unavailable_paltforms
+
+    mocker.patch.object(plugin_config, "bison_use_browser", False)
+    mocker.patch.dict(unavailable_paltforms, {"bilibili": "需要启用 bison_use_browser"})
+
+    async with app.test_matcher(add_sub_matcher) as ctx:
+        bot = ctx.create_bot()
+        event_1 = fake_group_message_event(
+            message=Message("添加订阅"),
+            sender=Sender(card="", nickname="test", role="admin"),
+            to_me=True,
+        )
+        ctx.receive_event(bot, event_1)
+        ctx.should_pass_rule()
+        ctx.should_call_send(
+            event_1,
+            BotReply.add_reply_on_platform(platform_manager=platform_manager, common_platform=common_platform),
+            True,
+        )
+        event_2 = fake_group_message_event(
+            message=Message("全部"), sender=Sender(card="", nickname="test", role="admin")
+        )
+        ctx.receive_event(bot, event_2)
+        ctx.should_rejected()
+        ctx.should_call_send(
+            event_2,
+            BotReply.add_reply_on_platform_input_allplatform(platform_manager),
+            True,
+        )
+        event_3 = fake_group_message_event(message=Message("bilibili"), sender=fake_admin_user)
+        ctx.receive_event(bot, event_3)
+        ctx.should_call_send(
+            event_3,
+            BotReply.add_reply_platform_unavailable("bilibili", "需要启用 bison_use_browser"),
+            True,
+        )

--- a/tests/utils.py
+++ b/tests/utils.py
@@ -146,6 +146,10 @@ class BotReply:
         extra_text = ("1." + target_promot + "\n2.") if target_promot else ""
         return extra_text + base_text
 
+    @staticmethod
+    def add_reply_platform_unavailable(platform: str, reason: str) -> str:
+        return f"无法订阅 {platform}，{reason}"
+
     add_reply_on_id_input_error = "id输入错误"
     add_reply_on_target_parse_input_error = "不能从你的输入中提取出id，请检查你输入的内容是否符合预期"
     add_reply_on_platform_input_error = "平台输入错误"

